fmt/std.h:143: multiple definition of `fmt::v12::detail::normalize_libcxx_inline_namespaces(fmt::v12::basic_string_view<char>, char*)';

Since the last commit, we got this error many time during link in our project:

`fmt/std.h:143: multiple definition of 'fmt::v12::detail::normalize_libcxx_inline_namespaces(fmt::v12::basic_string_view<char>, char*)';`

Everything was working before. This error seems directly related to the latest fmt commit.

--- a/include/fmt/std.h
+++ b/include/fmt/std.h
@@ -139,8 +139,8 @@ template <typename Variant, typename Char> class is_variant_formattable {
 #endif  // FMT_CPP_LIB_VARIANT
 
 #if FMT_USE_RTTI
-inline string_view normalize_libcxx_inline_namespaces(string_view demangled_name_view,
-                                               char* begin) {
+inline string_view normalize_libcxx_inline_namespaces(
+    string_view demangled_name_view, char* begin) {
   // Normalization of stdlib inline namespace names.
   // libc++ inline namespaces.
   //  std::__1::*       -> std::*

